### PR TITLE
fix: staging github integration

### PR DESCRIPTION
**Pull request by bigglesandginger**
# Description

Changes required to get GitHub integration working.
Instructions as to how to administer Keycloak (required to provide users that register with the `user` group to access the data)

Resolves #(issue)

### CODE REVIEW

File: kubernetes/staging/gateway-keycloak-deployment.yaml [hunk 1]
```diff
@@ -58,6 +58,9 @@ spec:
               value: app.staging.terarium.ai
             - name: KC_HOSTNAME_PORT
               value: "443"
+            - name: KC_HOSTNAME_ADMIN_URL
+              #value: https://app.staging.terarium.ai/auth
+              value: https://localhost:8443
```
Comment (YohannParis) [inline kubernetes/staging/gateway-keycloak-deployment.yaml:63]: This was the issue were `localhost:8443` is hard coded within Keycloack database?

Comment (bigglesandginger) [reply kubernetes/staging/gateway-keycloak-deployment.yaml:63]: No, rather the issue was/is that it is impossible as configured for the Admin UI to be accessible from the outside, thus localhost is the only way to make this work.  Now this is most likely an SSL/non-SSL issue at various levels but after 3 days of horsing around, I found that this works all the time, and is significantly more secure, so I think this is the solution we should go with, at least for now.

File: kubernetes/staging/administrate-staging-keycloak/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+| CURRENT | NAME | CLUSTER | AUTHINFO | NAMESPACE |
+| --- | --- | --- | --- | --- |
+| * | askem-staging | askem-staging | kubernetes-admin | terarium |
+| | docker-desktop | docker-desktop | docker-desktop | |
+
+To change the current context run `./admcloak.sh set docker-desktop` to switch from `askem-staging` to `docker-desktop`
+
+### SSH Configuration
+
+Add the following forward (`LocalForward 16443 172.16.60.226:6443`) to your SSH rule for `Host uncharted-askem-prod-askem-staging-kube-manager-1`
```
Comment (dgauldie) [inline kubernetes/staging/administrate-staging-keycloak/README.md:22]: should this information be put into an open source readme?

Comment (bigglesandginger) [reply kubernetes/staging/administrate-staging-keycloak/README.md:22]: Great question.  This information is about an IP address that is inaccesible from the Internet, but if a bad actor got inside then in theory they could know this IP...

**Pull request closed without merging**